frontend/src/features/documents/list/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { Button } from 'components/common/buttons/Button';
+import { InlineForm, InlineInput } from 'components/common/form/styles';
+import { Scrollable as ScrollableBase } from 'components/common/Scrollable/Scrollable';
+import styled from 'styled-components';
+
+export const ListPage = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width: 100%;
+  gap: 2.5rem;
+  padding: 0;
+`;
+
+export const Scrollable = styled(ScrollableBase)`
+  padding: 1.6rem 3.2rem;
+  width: 100%;
+`;
+
+export const PageHeader = styled.h3`
+  text-align: left;
+`;
+
+export const PageToolbar = styled.div`
+  align-items: center;
+  padding: 0;
+  padding-bottom: 2rem;
+`;
+
+export const Spacer = styled.div`
+  display: flex;
+  flex: 1 1 auto;
+`;
+
+export const FileIcon = styled(Button)`
+  &&.btn {
+    background-color: #fff;
+    color: ${({ theme, disabled }) =>
+      disabled ? theme.css.disabledColor : theme.css.primaryColor};
```

Comment:
I would push back on the extra customization for the button. We should just use the standard button with maybe a variation.
@devinleighsmith  thoughts?

Reply:
Removed this style since it was not needed for now, also did some clean up that was not needed for now